android/app/build.gradle

```diff
@@ -41,6 +45,36 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    if (dropboxAppID) {
+        def dropboxActivity = """<activity
+            android:name="com.dropbox.core.android.AuthActivity"
+            android:configChanges="orientation|keyboard"
+            android:launchMode="singleTask">
+          <intent-filter>
+            <data android:scheme="db-${dropboxAppID}" />
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.BROWSABLE" />
+            <category android:name="android.intent.category.DEFAULT" />
+          </intent-filter>
+        </activity>""";
+
+
+        applicationVariants.all { variant ->
+            variant.outputs.each { output ->
+                output.processManifest.doLast {
+                    File manifestOutFile = new File(output.processManifest.manifestOutputDirectory, "AndroidManifest.xml")
+                    if (!manifestOutFile.isFile()) {
+                        manifestOutFile = new File(new File(output.processManifest.manifestOutputDirectory, output.dirName),"AndroidManifest.xml")
+                    }
```

Comment:
@hristoterezov, why are you doing the computation of `manifestOutFile`? Isn't there a `manifestOutputFile` property already (like `manifestOutputDirectory`)?

Reply:
There was but If I have understand it correctly it is deprecated and doesn't exist anymore.